usecase/useCase_pokemons.go

```diff
@@ -0,0 +1,124 @@
+package usecases
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
```

Comment:
For best practice separate the import libraries like:

Import(
// Standar library packages

// Our packages

// Third party libraries

)